dashboard/src/views/settings/AccountTeam.vue

```diff
@@ -18,20 +26,87 @@
 					</div>
 				</template>
 			</ListItem>
+
+			<Dialog
+				:options="{ title: 'Manage Team' }"
+				v-model="showManageTeamDialog"
+			>
+				<template v-slot:body-content>
+					<ListItem
+						v-for="member in $account.child_team_members"
+						:title="`${member.name}`"
+						:description="member.name"
+						:key="member.name"
+					>
+						<template #actions>
+							<ErrorMessage :message="$resourceErrors" />
+							<Button
+								class="ml-2 p-4"
+								@click="$resources.removeMember.submit({ child_team: member.name })"
+								:loading="$resources.removeMember.loading"
+							>
+								Remove
+							</Button>
+						</template>
+					</ListItem>
+					<div v-if="showManageTeamForm">
+						<h5 class="mt-5 text-sm font-semibold"> Create child team</h5>
+						<Input
+							label="Enter the team Id of child team."
```

Comment:
'Enter the email to create a new child team for shared access'

Let's be a little more explicit here